students/17th/team1/gyuboshim/westagram/user/views.py

```diff
@@ -0,0 +1,68 @@
+from django.shortcuts import render
+
+import json
+from django.views   import View
+from django.http    import HttpResponse, JsonResponse
+
+from .models        import User
+
+PASSWORD_LEN_LIMIT = 8
+
+class UserView(View):
+    def post(self, request):
+        try:
+            data            =   json.loads(request.body)
+            user_data       =   data['user']
+            phone_number    =   user_data['phone_number']
+            email_adress    =   user_data['email_adress']
+            name            =   user_data['name']
+            nickname        =   user_data['nickname']
+            password        =   user_data['password']
+
+
+            if user_data['email_adress'] and user_data['password']:
```

Comment:
이 로직이 바깥에 위치하지 않고 같은 인덴트에 위치해도~ 기대 효과대로 동작할것 같습니다. 
굳이 위에 변수로 다 담아 놓고 여기서 다시 딕셔너리의 키를 찍는 이유를 알고 싶습니다. 
```suggestion
            if not email_adress and not password:
                return JsonResponse('.......') 
```
이렇게 처리하면 위의 중첩 if문이 필요 없지 않을까요?

Reply:
이 부분 변수로 모두 고쳤습니다! 저도 어제 푸쉬하고 다시보면서 변수선언하고 변수를 안썻네..라는 상황을 마주해서 수정했습니다!

Reply:
중첩 if문이 필요가 없어질 수 있다는 부분은 이해가 잘 되지않습니다..ㅜ 일단 조금 더 수정한 상태이니 한번 리뷰 부탁드립니다

Reply:
그냥 컨셉의 문제 같습니다. 이값과 이값이 들어와야만 아래에 있는 일을 하겠어와, 이것 이것 이것이 들어오지 않으면 일단 에러 리턴을 받으므로 더이상 진행하지 않는것 제가 제안드린건 이런  이야기 였습니다.